Linked List Questions/linkedListQ1.java

```diff
@@ -71,7 +71,6 @@ static SinglyLinkedListNode insertNodeAtPosition(SinglyLinkedListNode head, int
         {
             atIndex++;
             currentNode = currentNode.next;
-            System.out.println(currentNode.data);
```

Comment:
This was added for debug before but we don't need it anymore

Reply:
this looks good